pysat/instruments/methods/general.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""Provides generalized routines for integrating instruments into pysat.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+import pandas as pds
+
+import pysat
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def list_files(tag=None, sat_id=None, data_path=None, format_str=None,
+               supported_tags=None, fake_daily_files_from_monthly=False,
+               two_digit_year_break=None):
+    """Return a Pandas Series of every file for chosen satellite data.
+
+    This routine provides a standard interfacefor pysat instrument modules.
+
+    Parameters
+    -----------
+    tag : (string or NoneType)
+        Denotes type of file to load.  Accepted types are <tag strings>.
+        (default=None)
+    sat_id : (string or NoneType)
+        Specifies the satellite ID for a constellation.  Not used.
+        (default=None)
+    data_path : (string or NoneType)
+        Path to data directory.  If None is specified, the value previously
+        set in Instrument.files.data_path is used.  (default=None)
+    format_str : (string or NoneType)
+        User specified file format.  If None is specified, the default
+        formats associated with the supplied tags are used. (default=None)
+    supported_tags : (dict or NoneType)
+        keys are sat_id, each containing a dict keyed by tag
+        where the values file format template strings. (default=None)
+    fake_daily_files_from_monthly : bool
+        Some CDAWeb instrument data files are stored by month, interfering
+        with pysat's functionality of loading by day. This flag, when true,
+        appends daily dates to monthly files internally. These dates are
+        used by load routine in this module to provide data by day.
+    two_digit_year_break : int
+        If filenames only store two digits for the year, then
+        '1900' will be added for years >= two_digit_year_break
+        and '2000' will be added for years < two_digit_year_break.
+
+    Returns
+    --------
+    pysat.Files.from_os : (pysat._files.Files)
```

Comment:
What's our style, I don't recall.  Parens or no parens?

Reply:
We use both depending on which file you look in.  Happy to use either, but I don't know if we've ever made a formal decision.

Reply:
From a quick survey of docstrings: about 25 files in this branch mostly use paren, about 21 mostly use no paren, not counting deprecated files or tests.

Reply:
I haven't been using parens

Reply:
We don't use parens in pysatModels, so let's ditch them as we go onwards.

Reply:
But honestly, for this pull it looks like you went with parens, so that's fine until we meet again.

Reply:
should be consistent now